### ISSUE TITLE
l10n-argentina vs odoo-argentina-ce

### ISSUE THREAD

**Issue by pauletti** [OCA/l10n-argentina#78]
Hello, I was searching for open source odoo Argentina l10n and ended up with two candidates:

- l10n-argentina (this one) 
- odoo-argentina-ce (https://github.com/ingadhoc/odoo-argentina-ce.git)

As far as I can tell, this one seems to follow better Coding Standards & Best Practices, but only have up to v14,
whereas odoo-argentina-ce goes up to v16.
On the other hand the developers of odoo-argentina-ce (adhoc) claim to  be the official maintainer of the l10n in the EE of Odoo.
To increase even more my confusion adhoc is referenced in certain code of l10n-argentina. 

To summarize, what I am politely trying to ask is that it would be nice if someone can clarify the relation (if any) between  l10n-argentina and odoo-argentina-ce. For two reasons:

- codes are very similar and seems that work is being duplicated, why not merge
- help users like me decide which one is the appropriate on for my needs

Thanks

**Comment by nimarosa** [comment]
Hello, 

There are a lot of different localizations for Argentina, not only the one you are listing. 

In this repository, we a trying to make the "oficial" one for CE. We want ADHOC and other localization developers to merge here and make a localization maintained by all the different partners in Argentina.  

The localization here, is a rewrite and a merge of all other localizations. Also there are other improvements here that are not in other private localizations and as you said, it follows Coding Standards and Best Practices. 

We are currently migrating to v16.0 and expect to have more modules soon here. 

This repository was abandoned a time ago, and recently we are trying to update it to have all localization modules here and collaborate between Argentina Partners. 

If you are starting with Odoo CE Argentinian localization, i suggest to use this one, since we expect to have all localization modules merged here soon.
